com.unity.robotics.urdf-importer/Runtime/UnityMeshImporter/Plugins/AssimpNet/AssimpNet.xml

```diff
@@ -3735,7 +3735,7 @@
         <member name="F:Assimp.DefaultLogStream.Debugger">
             <summary>
             MSVC only: Stream the log to the debugger (this relies
-            on OutputDebugString from the Win32 SDK).
+            on outputDebugString from the Win32 SDK).
```

Comment:
revert all modifications made to AssimpNet.xml. (grab the original unchanged one from "dev" branch?)